### ISSUE TITLE
Is it possible to disable the aggregation "Sum(..)" shown in the queryName attribute. 

### ISSUE THREAD

**Issue by mgam1** [Microsoft/PowerBI-visuals#122]
Hello, 

Description of data:
I have a number of attributes (attr1...attr8) that makeup a Dot object and I have a number of categories known as Children. There can be several dots that belong to a specific category. In my source data I have a table describing which dots belong to which categories. 

Capabilities:
Here is a image of my data roles and mappings. You can see that I've written a groupby in my data mappings, this was an attempt to group the Dot attributes belonging to a specific category.  
![image](https://cloud.githubusercontent.com/assets/13663012/10339867/510410aa-6d05-11e5-93e8-f3ec5fc8bee5.png)

Issue: 
Looking at the dataview returned, I can see that the dots belonging to the same category have had their attributes summed with each other. for example attr1 of every dot belonging to the same category is summed together. This is preventing me from seeing the individual values that make up each dot.  

In short I would like to know if there is anyway to stop the Sum aggregation shown by the queryName attribute below from occurring, and instead have the individual values for the attributes returned.

![image](https://cloud.githubusercontent.com/assets/13663012/10339379/e5399f68-6d02-11e5-856a-f74c1cf04290.png)

Thanks
